src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -731,13 +751,51 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
                 (_evaluationStep, _) = AddEvaluationStep(_softMaxTensor, _labelTensor);
                 _softmaxTensorName = _softMaxTensor.name;
                 _predictionTensorName = _prediction.name;
+
+                // Add annotations as key values, if they exist.
+                VBuffer<ReadOnlyMemory<char>> keysVBuffer = default;
+                if (inputSchema[labelColumnName].HasKeyValues())
+                {
+                    inputSchema[labelColumnName].GetKeyValues(ref keysVBuffer);
+                    _keyValueAnnotations = keysVBuffer.DenseValues().ToArray();
+                }
```

Comment:
If it does not have key values then just add numeric values (0 - classCount -1) as string to have something in there